src/Database/DriverInterface.php

```diff
@@ -151,10 +151,10 @@ public function supportsSavePoints(): bool;
      * Returns a value in a safe representation to be used in a query string
      *
      * @param mixed $value The value to quote.
-     * @param int $type Type to be used for determining kind of quoting to perform.
+     * @param int $type Must be one of the \PDO::PARAM_* constants
      * @return string
      */
-    public function quote($value, $type): string;
+    public function quote($value, int $type): string;
```

Comment:
This type hint was missing, but don't know if it's too late.

Reply:
This is a medium risk change as there could be implementations of DriverInterface in the wild. I don't think there will be many though.

Reply:
I can revert it. Honestly, the $value parameter should be `string` the same as PDO::quote() so maybe we just leave both alone until 5.